src/store/RelayPendingQueryTracker.js

```diff
@@ -118,12 +118,14 @@ class PendingFetch {
   _errors: Array<?Error>;
 
   constructor(
+    /* eslint-disable comma-dangle */
```

Comment:
At first I thought it was an babel-eslint issue but I think the `comma-dangle` rule may have to be rewritten (maybe through https://github.com/babel/eslint-plugin-babel). The `isMultiline` doesn't account for flow types.

![screen shot 2015-12-08 at 12 08 21 pm](https://cloud.githubusercontent.com/assets/588473/11662398/6a70a25e-9da4-11e5-9304-973488290971.png)

So it looks like a multline which causes the error (without the flow type it's fine since it starts/ends on the same line without it)